packages/expo-auth-session/src/AuthRequest.types.ts

```diff
@@ -97,6 +97,11 @@ export type AuthRequestPromptOptions = Omit<WebBrowserOpenOptions, 'windowFeatur
    * @platform web
    */
   windowFeatures?: WebBrowserWindowFeatures;
+  /**
+   * Flag to indicate that the session should be in ephemeral mode.
+   * @platform ios
+   */
+  preferEphemeralSession?: boolean;
```

Comment:
Hmm, maybe we could leverage the `AuthSessionOpenOptions` type instead:

https://github.com/thorbenprimke/expo/blob/740d9407ffa6108a691d388ddeac1310191bd265/packages/expo-web-browser/src/WebBrowser.types.ts#L91

So instead of adding the option here, we can do this change on line 77:
```diff
diff --git a/packages/expo-auth-session/src/AuthRequest.types.ts b/packages/expo-auth-session/src/AuthRequest.types.ts
index 851610f3a4..1b26f0890d 100644
--- a/packages/expo-auth-session/src/AuthRequest.types.ts
+++ b/packages/expo-auth-session/src/AuthRequest.types.ts
@@ -77,1 +77,1 @@ export enum Prompt {
-export type AuthRequestPromptOptions = Omit<WebBrowserOpenOptions, 'windowFeatures'> & {
+export type AuthRequestPromptOptions = Omit<AuthSessionOpenOptions, 'windowFeatures'> & {
```

Then you wouldn't need to redefine the `preferEphemeralSession` flag, what do you think?

Reply:
@barthap - Yes, I think that would be better. Also would avoid future changes if other parameter are added to `AuthSessionOpenOptions`. 

Let me update the PR then.